src/clients/Deluge.ts

```diff
@@ -203,6 +218,11 @@ export default class Deluge implements TorrentClient {
 				await this.call<void>("label.add", [label]);
 				await this.call<void>("label.set_torrent", [infoHash, label]);
 			}
+		} catch (e) {
+			logger.warn({
+				label: Label.DELUGE,
+				message: `Failed to label ${infoHash} as ${label}`,
```

Comment:
Having moved this from inject() to setLabel() seems to have reduced the error (warning) messaging to not include the `newTorrent.name` - while I don't think its particularly likely to ever be necessary for debugging in a position that we can't still use the infoHash or its position in the logs, I do wonder if there's a reason we shouldn't just pass the newTorrent.name to setLabel as well for more readability in the logs for users.

Reply:
I had the same thought process about having to pass in the name, but it's not hard to do. I'll make the change